distributed/cli/utils.py

```diff
@@ -22,6 +22,30 @@ async def cleanup_and_stop():
                 loop.stop()
 
         loop.add_callback_from_signal(cleanup_and_stop)
+
+        # Restore old signal handler to allow for a quicker exit
+        # if the user sends the signal again.
+        signal.signal(sig, old_handlers[sig])
+
+    for sig in [signal.SIGINT, signal.SIGTERM]:
+        old_handlers[sig] = signal.signal(sig, handle_signal)
+
+
+def install_signal_handlers2(loop=None, cleanup=None):
```

Comment:
This is only temporary since I need to support `install_signal_handlers` until I have adjusted `dask_scheduler` as well.